lms/static/js/spec/main.js

```diff
@@ -93,11 +93,11 @@
             // Discussion classes loaded explicitly until they are converted to use RequireJS
             'DiscussionModuleView': 'xmodule_js/common_static/coffee/src/discussion/discussion_module_view',
 
-            'js/bookmarks/collections/bookmarks': 'js/bookmarks/collections/bookmarks',
-            'js/bookmarks/models/bookmark': 'js/bookmarks/models/bookmark',
-            'js/bookmarks/views/bookmarks_list_button': 'js/bookmarks/views/bookmarks_list_button',
-            'js/bookmarks/views/bookmarks_list': 'js/bookmarks/views/bookmarks_list',
-            'js/bookmarks/views/bookmark_button': 'js/bookmarks/views/bookmark_button',
+            'js/courseware/collections/bookmarks': 'js/courseware/collections/bookmarks',
```

Comment:
Sorry if this wasn't clear, but I didn't mean that all the bookmarks code should be moved into courseware. I think it is clearer to leave everything but the factory underneath ``js/bookmarks``, and just move the factory into courseware since it is now handles more than just bookmarks.